src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1551,7 +1536,7 @@ protected override void ProcessRecord()
         }
 
         /// <summary>
-        /// Implementing ^C, after start the BeginGetResponse.
+        /// To implement ^C.
```

Comment:
I don't see value in the change. Please revert.

Reply:
BeginGetResponse doesn't exist (It was there in PS5 but was since removed).
I think we should change the comment

Reply:
I see such comment is used in many places so I am agreed.